src/seedsigner/views/seed_views.py

```diff
@@ -2001,10 +2001,31 @@ def __init__(self):
         self.derivation_path = data.get("derivation_path")
```

Comment:
@kdmukai merge of PR #443 broke this PR (or maybe I did when trying to resolve the simple conflict).

I believe adding back `self.seed_num = data.get("seed_num")` will resolve the issue.